item/Tyler Stokes; Belzor's Fantastical Flora.json

```diff
@@ -0,0 +1,2749 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "BFF",
```

Comment:
you need to use a longer `json`/`source`; consider e.g. `"BelzorsFantasticalFlora"`